testcases/open_posix_testsuite/conformance/interfaces/strchr/1-1.c

```diff
@@ -0,0 +1,103 @@
+/*
+ * This file is licensed under the GPL license. For the full content
+ * of this license, see the COPYING file at the top level of this
+ * source tree.
+ */
```

Comment:
This is copy paste from rest of the code. It should state GPL2 or any other. How about using SPDX identifier (at the top)?

`// SPDX-License-Identifier: GPL-2.0-or-later`